### ISSUE TITLE
BatteryTotalChargeEnergy and BatteryTotalDischargeEnergy are missing unit conversion

### ISSUE THREAD

**Issue by Kobe-Wulteputte** [hultenvp/solis-sensor#155]
BatteryTotalChargeEnergy and distcharge energy are send as MWh, but read as kWh.

I checked the api calls and I got the following response:
```
...'batteryTotalChargeEnergy': 1.811, 'batteryTotalChargeEnergyStr': 'MWh' ...
```

The entity in HA now shows 1.81 kWh, so this should be * 1000 and with a precision of 3

I am using soliscloud.com

**Comment by hultenvp** [comment]
Looked in the code and this one indeed has no correction for changing units (solicloud changes the unit depending on the value, HA requires constant unit).

**Comment by hultenvp** [comment]
Released 2.3.4. Cannot test as I have no hybrid inverter, but should now convert MWh to kWh
